### ISSUE TITLE
Navigation Block: Block is broken when tabs on the block sidebar are moved with the keyboard

### ISSUE THREAD

**Issue by t-hamano** [WordPress/gutenberg#52691]
### Description

This is a very strange bug. When you select a Navigation block, tabs List View, Settings, and Styles appear in the block sidebar. If I press the `Tab` key while the ListView tab has focus, the block is broken.

- This does not occur in other blocks with tabs.
- If the navigation block is inserted in the post editor instead of the site editor, the problem does not seem to occur.

The following message will be output to the browser's error log.

<details><summary>Details</summary>

```
react-dom.js?ver=18:14897 Uncaught Error: Objects are not valid as a React child (found: object with keys {rendered}). If you meant to render a collection of children, use an array instead.
    at throwOnInvalidObjectType (react-dom.js?ver=18:14897:11)
    at createChild (react-dom.js?ver=18:15149:9)
    at reconcileChildrenArray (react-dom.js?ver=18:15414:27)
    at reconcileChildFibers (react-dom.js?ver=18:15831:18)
    at reconcileChildren (react-dom.js?ver=18:19177:30)
    at updateHostComponent (react-dom.js?ver=18:19934:5)
    at beginWork (react-dom.js?ver=18:21628:16)
    at HTMLUnknownElement.callCallback (react-dom.js?ver=18:4151:16)
    at Object.invokeGuardedCallbackDev (react-dom.js?ver=18:4200:18)
    at invokeGuardedCallback (react-dom.js?ver=18:4264:33)
```

```
The above error occurred in the <div> component:

    at div
    at div
    at MotionComponent (http://localhost:8888/wp-content/plugins/gutenberg/build/components/index.min.js?ver=8fa906308eea7a7e3625:88948:65)
    at http://localhost:8888/wp-content/plugins/gutenberg/build/components/index.min.js?ver=8fa906308eea7a7e3625:63792:10
    at StyleProvider (http://localhost:8888/wp-content/plugins/gutenberg/build/components/index.min.js?ver=8fa906308eea7a7e3625:69711:5)
    at Fill (http://localhost:8888/wp-content/plugins/gutenberg/build/components/index.min.js?ver=8fa906308eea7a7e3625:68211:3)
    at Fill
    at UnforwardedPopover (http://localhost:8888/wp-content/plugins/gutenberg/build/components/index.min.js?ver=8fa906308eea7a7e3625:63822:5)
    at button
    at Tooltip (http://localhost:8888/wp-content/plugins/gutenberg/build/components/index.min.js?ver=8fa906308eea7a7e3625:73847:5)
    at UnforwardedButton (http://localhost:8888/wp-content/plugins/gutenberg/build/components/index.min.js?ver=8fa906308eea7a7e3625:42939:7)
    at div
    at UnconnectedDropdown (http://localhost:8888/wp-content/plugins/gutenberg/build/components/index.min.js?ver=8fa906308eea7a7e3625:50646:68)
    at UnconnectedDropdownMenu (http://localhost:8888/wp-content/plugins/gutenberg/build/components/index.min.js?ver=8fa906308eea7a7e3625:50311:68)
    at NavigationMenuSelector (http://localhost:8888/wp-content/plugins/gutenberg/build/block-library/index.min.js?ver=321c5fe8e4395e5589cc:37955:3)
    at div
    at http://localhost:8888/wp-content/plugins/gutenberg/build/components/index.min.js?ver=8fa906308eea7a7e3625:751:66
    at UnconnectedHStack (http://localhost:8888/wp-content/plugins/gutenberg/build/components/index.min.js?ver=8fa906308eea7a7e3625:55483:71)
    at div
    at UnforwardedPanelBody (http://localhost:8888/wp-content/plugins/gutenberg/build/components/index.min.js?ver=8fa906308eea7a7e3625:63340:5)
    at ToolsPanelInspectorControl (http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:42005:3)
    at StyleProvider (http://localhost:8888/wp-content/plugins/gutenberg/build/components/index.min.js?ver=8fa906308eea7a7e3625:69711:5)
    at Fill (http://localhost:8888/wp-content/plugins/gutenberg/build/components/index.min.js?ver=8fa906308eea7a7e3625:68211:3)
    at Fill
    at InspectorControlsListViewFill
    at StyleProvider (http://localhost:8888/wp-content/plugins/gutenberg/build/components/index.min.js?ver=8fa906308eea7a7e3625:69711:5)
    at InspectorControlsFill (http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:41967:3)
    at MenuInspectorControls (http://localhost:8888/wp-content/plugins/gutenberg/build/block-library/index.min.js?ver=321c5fe8e4395e5589cc:37749:5)
    at RecursionProvider (http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:49142:3)
    at EntityProvider (http://localhost:8888/wp-content/plugins/gutenberg/build/core-data/index.min.js?ver=7e620019e1f29c5fa0de:2623:3)
    at Navigation (http://localhost:8888/wp-content/plugins/gutenberg/build/block-library/index.min.js?ver=321c5fe8e4395e5589cc:36761:3)
    at <anonymous> (http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:26856:9)
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:26826:79
    at Edit (http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:14207:5)
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:58172:11
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:58379:96
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:59865:113
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:62372:11
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:62821:116
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:60337:102
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:61415:11
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:59614:65
    at http://localhost:8888/wp-content/plugins/gutenberg/build/edit-site/index.min.js?ver=a6818390dbd946f542fa:26279:28
    at http://localhost:8888/wp-content/plugins/gutenberg/build/edit-site/index.min.js?ver=a6818390dbd946f542fa:26381:5
    at http://localhost:8888/wp-content/plugins/gutenberg/build/edit-site/index.min.js?ver=a6818390dbd946f542fa:26000:5
    at FilteredComponentRenderer (http://localhost:8888/wp-content/plugins/gutenberg/build/components/index.min.js?ver=8fa906308eea7a7e3625:56191:9)
    at EditWithGeneratedProps (http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:14227:5)
    at BlockEdit (http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:14304:5)
    at BlockCrashBoundary (http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:15281:5)
    at BlockListBlock (http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:15689:10)
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:58222:5
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:62394:5
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:59073:5
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:59450:5
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:60806:89
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:62838:124
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:60409:79
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:61437:5
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:61504:5
    at FilteredComponentRenderer (http://localhost:8888/wp-content/plugins/gutenberg/build/components/index.min.js?ver=8fa906308eea7a7e3625:56191:9)
    at http://localhost:8888/wp-content/plugins/gutenberg/build/compose/index.min.js?ver=226d195c88c749c2de26:2647:10
    at http://localhost:8888/wp-content/plugins/gutenberg/build/data/index.min.js?ver=d606b1a3c842ef80e38c:2057:82
    at http://localhost:8888/wp-content/plugins/gutenberg/build/data/index.min.js?ver=d606b1a3c842ef80e38c:2185:77
    at shouldComponentUpdate (http://localhost:8888/wp-content/plugins/gutenberg/build/compose/index.min.js?ver=226d195c88c749c2de26:2787:12)
    at Items (http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:16216:3)
    at BlockListItems
    at BlockContextProvider (http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:13462:3)
    at UncontrolledInnerBlocks (http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:36580:5)
    at div
    at GroupEdit (http://localhost:8888/wp-content/plugins/gutenberg/build/block-library/index.min.js?ver=321c5fe8e4395e5589cc:23401:3)
    at Edit (http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:14207:5)
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:58172:11
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:58379:96
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:59865:113
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:62372:11
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:62821:116
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:60337:102
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:61415:11
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:59614:65
    at http://localhost:8888/wp-content/plugins/gutenberg/build/edit-site/index.min.js?ver=a6818390dbd946f542fa:26279:28
    at http://localhost:8888/wp-content/plugins/gutenberg/build/edit-site/index.min.js?ver=a6818390dbd946f542fa:26381:5
    at http://localhost:8888/wp-content/plugins/gutenberg/build/edit-site/index.min.js?ver=a6818390dbd946f542fa:26000:5
    at FilteredComponentRenderer (http://localhost:8888/wp-content/plugins/gutenberg/build/components/index.min.js?ver=8fa906308eea7a7e3625:56191:9)
    at EditWithGeneratedProps (http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:14227:5)
    at BlockEdit (http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:14304:5)
    at BlockCrashBoundary (http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:15281:5)
    at BlockListBlock (http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:15689:10)
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:58222:5
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:62394:5
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:59073:5
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:59450:5
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:60806:89
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:62838:124
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:60409:79
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:61437:5
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:61504:5
    at FilteredComponentRenderer (http://localhost:8888/wp-content/plugins/gutenberg/build/components/index.min.js?ver=8fa906308eea7a7e3625:56191:9)
    at http://localhost:8888/wp-content/plugins/gutenberg/build/compose/index.min.js?ver=226d195c88c749c2de26:2647:10
    at http://localhost:8888/wp-content/plugins/gutenberg/build/data/index.min.js?ver=d606b1a3c842ef80e38c:2057:82
    at http://localhost:8888/wp-content/plugins/gutenberg/build/data/index.min.js?ver=d606b1a3c842ef80e38c:2185:77
    at shouldComponentUpdate (http://localhost:8888/wp-content/plugins/gutenberg/build/compose/index.min.js?ver=226d195c88c749c2de26:2787:12)
    at Items (http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:16216:3)
    at BlockListItems
    at BlockContextProvider (http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:13462:3)
    at UncontrolledInnerBlocks (http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:36580:5)
    at div
    at GroupEdit (http://localhost:8888/wp-content/plugins/gutenberg/build/block-library/index.min.js?ver=321c5fe8e4395e5589cc:23401:3)
    at Edit (http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:14207:5)
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:58172:11
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:58379:96
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:59865:113
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:62372:11
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:62821:116
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:60337:102
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:61415:11
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:59614:65
    at http://localhost:8888/wp-content/plugins/gutenberg/build/edit-site/index.min.js?ver=a6818390dbd946f542fa:26279:28
    at http://localhost:8888/wp-content/plugins/gutenberg/build/edit-site/index.min.js?ver=a6818390dbd946f542fa:26381:5
    at http://localhost:8888/wp-content/plugins/gutenberg/build/edit-site/index.min.js?ver=a6818390dbd946f542fa:26000:5
    at FilteredComponentRenderer (http://localhost:8888/wp-content/plugins/gutenberg/build/components/index.min.js?ver=8fa906308eea7a7e3625:56191:9)
    at EditWithGeneratedProps (http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:14227:5)
    at BlockEdit (http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:14304:5)
    at BlockCrashBoundary (http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:15281:5)
    at BlockListBlock (http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:15689:10)
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:58222:5
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:62394:5
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:59073:5
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:59450:5
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:60806:89
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:62838:124
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:60409:79
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:61437:5
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:61504:5
    at FilteredComponentRenderer (http://localhost:8888/wp-content/plugins/gutenberg/build/components/index.min.js?ver=8fa906308eea7a7e3625:56191:9)
    at http://localhost:8888/wp-content/plugins/gutenberg/build/compose/index.min.js?ver=226d195c88c749c2de26:2647:10
    at http://localhost:8888/wp-content/plugins/gutenberg/build/data/index.min.js?ver=d606b1a3c842ef80e38c:2057:82
    at http://localhost:8888/wp-content/plugins/gutenberg/build/data/index.min.js?ver=d606b1a3c842ef80e38c:2185:77
    at shouldComponentUpdate (http://localhost:8888/wp-content/plugins/gutenberg/build/compose/index.min.js?ver=226d195c88c749c2de26:2787:12)
    at Items (http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:16216:3)
    at BlockListItems
    at BlockContextProvider (http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:13462:3)
    at UncontrolledInnerBlocks (http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:36580:5)
    at ControlledInnerBlocks (http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:36642:72)
    at header
    at TemplatePartInnerBlocks (http://localhost:8888/wp-content/plugins/gutenberg/build/block-library/index.min.js?ver=321c5fe8e4395e5589cc:63311:11)
    at RecursionProvider (http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:49142:3)
    at TemplatePartEdit (http://localhost:8888/wp-content/plugins/gutenberg/build/block-library/index.min.js?ver=321c5fe8e4395e5589cc:63163:3)
    at Edit (http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:14207:5)
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:58172:11
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:58379:96
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:59865:113
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:62372:11
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:62821:116
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:60337:102
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:61415:11
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:59614:65
    at http://localhost:8888/wp-content/plugins/gutenberg/build/edit-site/index.min.js?ver=a6818390dbd946f542fa:26279:28
    at http://localhost:8888/wp-content/plugins/gutenberg/build/edit-site/index.min.js?ver=a6818390dbd946f542fa:26381:5
    at http://localhost:8888/wp-content/plugins/gutenberg/build/edit-site/index.min.js?ver=a6818390dbd946f542fa:26000:5
    at FilteredComponentRenderer (http://localhost:8888/wp-content/plugins/gutenberg/build/components/index.min.js?ver=8fa906308eea7a7e3625:56191:9)
    at EditWithGeneratedProps (http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:14227:5)
    at BlockEdit (http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:14304:5)
    at BlockCrashBoundary (http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:15281:5)
    at BlockListBlock (http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:15689:10)
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:58222:5
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:62394:5
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:59073:5
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:59450:5
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:60806:89
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:62838:124
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:60409:79
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:61437:5
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:61504:5
    at FilteredComponentRenderer (http://localhost:8888/wp-content/plugins/gutenberg/build/components/index.min.js?ver=8fa906308eea7a7e3625:56191:9)
    at http://localhost:8888/wp-content/plugins/gutenberg/build/compose/index.min.js?ver=226d195c88c749c2de26:2647:10
    at http://localhost:8888/wp-content/plugins/gutenberg/build/data/index.min.js?ver=d606b1a3c842ef80e38c:2057:82
    at http://localhost:8888/wp-content/plugins/gutenberg/build/data/index.min.js?ver=d606b1a3c842ef80e38c:2185:77
    at shouldComponentUpdate (http://localhost:8888/wp-content/plugins/gutenberg/build/compose/index.min.js?ver=226d195c88c749c2de26:2787:12)
    at Items (http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:16216:3)
    at BlockListItems
    at div
    at Root (http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:16131:3)
    at BlockToolsBackCompat (http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:23207:3)
    at BlockList (http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:16207:81)
    at StyleProvider (http://localhost:8888/wp-content/plugins/gutenberg/build/components/index.min.js?ver=8fa906308eea7a7e3625:69711:5)
    at body
    at iframe
    at Iframe (http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:34749:3)
    at IframeIfReady (http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:34888:83)
    at EditorCanvas (http://localhost:8888/wp-content/plugins/gutenberg/build/edit-site/index.min.js?ver=a6818390dbd946f542fa:4748:3)
    at div
    at Resizable (http://localhost:8888/wp-content/plugins/gutenberg/build/components/index.min.js?ver=8fa906308eea7a7e3625:24799:28)
    at UnforwardedResizableBox (http://localhost:8888/wp-content/plugins/gutenberg/build/components/index.min.js?ver=8fa906308eea7a7e3625:66484:3)
    at ResizableEditor (http://localhost:8888/wp-content/plugins/gutenberg/build/edit-site/index.min.js?ver=a6818390dbd946f542fa:5364:3)
    at div
    at BlockTools (http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:23728:3)
    at SlotComponent (http://localhost:8888/wp-content/plugins/gutenberg/build/components/index.min.js?ver=8fa906308eea7a7e3625:68970:5)
    at Slot
    at http://localhost:8888/wp-content/plugins/gutenberg/build/components/index.min.js?ver=8fa906308eea7a7e3625:68744:3
    at EditSiteEditorCanvasContainerSlotSlot
    at SiteEditorCanvas (http://localhost:8888/wp-content/plugins/gutenberg/build/edit-site/index.min.js?ver=a6818390dbd946f542fa:5545:67)
    at BlockRefsProvider (http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:48559:3)
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:48615:5
    at http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:48984:5
    at WithRegistryProvider(Component)
    at DefaultBlockEditorProvider (http://localhost:8888/wp-content/plugins/gutenberg/build/edit-site/index.min.js?ver=a6818390dbd946f542fa:5167:3)
    at BlockEditor (http://localhost:8888/wp-content/plugins/gutenberg/build/edit-site/index.min.js?ver=a6818390dbd946f542fa:4907:80)
    at div
    at NavigableRegion (http://localhost:8888/wp-content/plugins/gutenberg/build/edit-site/index.min.js?ver=a6818390dbd946f542fa:31249:3)
    at div
    at div
    at div
    at InterfaceSkeleton (http://localhost:8888/wp-content/plugins/gutenberg/build/edit-site/index.min.js?ver=a6818390dbd946f542fa:31003:3)
    at BlockContextProvider (http://localhost:8888/wp-content/plugins/gutenberg/build/block-editor/index.min.js?ver=a7d22100e536e90c3456:13462:3)
    at EntityProvider (http://localhost:8888/wp-content/plugins/gutenberg/build/core-data/index.min.js?ver=7e620019e1f29c5fa0de:2623:3)
    at EntityProvider (http://localhost:8888/wp-content/plugins/gutenberg/build/core-data/index.min.js?ver=7e620019e1f29c5fa0de:2623:3)
    at Editor (http://localhost:8888/wp-content/plugins/gutenberg/build/edit-site/index.min.js?ver=a6818390dbd946f542fa:6557:3)
    at div
    at MotionComponent (http://localhost:8888/wp-content/plugins/gutenberg/build/components/index.min.js?ver=8fa906308eea7a7e3625:88948:65)
    at div
    at MotionComponent (http://localhost:8888/wp-content/plugins/gutenberg/build/components/index.min.js?ver=8fa906308eea7a7e3625:88948:65)
    at Resizable (http://localhost:8888/wp-content/plugins/gutenberg/build/components/index.min.js?ver=8fa906308eea7a7e3625:24799:28)
    at UnforwardedResizableBox (http://localhost:8888/wp-content/plugins/gutenberg/build/components/index.min.js?ver=8fa906308eea7a7e3625:66484:3)
    at ResizableFrame (http://localhost:8888/wp-content/plugins/gutenberg/build/edit-site/index.min.js?ver=a6818390dbd946f542fa:15924:3)
    at ErrorBoundary (http://localhost:8888/wp-content/plugins/gutenberg/build/edit-site/index.min.js?ver=a6818390dbd946f542fa:6720:5)
    at div
    at MotionComponent (http://localhost:8888/wp-content/plugins/gutenberg/build/components/index.min.js?ver=8fa906308eea7a7e3625:88948:65)
    at div
    at div
    at div
    at Layout (http://localhost:8888/wp-content/plugins/gutenberg/build/edit-site/index.min.js?ver=a6818390dbd946f542fa:12802:100)
    at RouterProvider (http://localhost:8888/wp-content/plugins/gutenberg/build/router/index.min.js?ver=0f50332f42b79121e4e2:183:3)
    at GlobalStylesProvider (http://localhost:8888/wp-content/plugins/gutenberg/build/edit-site/index.min.js?ver=a6818390dbd946f542fa:7320:3)
    at SlotFillProvider (http://localhost:8888/wp-content/plugins/gutenberg/build/components/index.min.js?ver=8fa906308eea7a7e3625:68402:3)
    at SlotFillProvider (http://localhost:8888/wp-content/plugins/gutenberg/build/components/index.min.js?ver=8fa906308eea7a7e3625:68825:5)
    at Provider (http://localhost:8888/wp-content/plugins/gutenberg/build/components/index.min.js?ver=8fa906308eea7a7e3625:68757:3)
    at div
    at ShortcutProvider (http://localhost:8888/wp-content/plugins/gutenberg/build/keyboard-shortcuts/index.min.js?ver=c19d0f8e6ae021685645:41:87)
    at App (http://localhost:8888/wp-content/plugins/gutenberg/build/edit-site/index.min.js?ver=a6818390dbd946f542fa:4598:67)

React will try to recreate this component tree from scratch using the error boundary you provided, BlockCrashBoundary.
```
</details> 

### Step-by-step reproduction instructions

N/A

### Screenshots, screen recording, code snippet

https://github.com/WordPress/gutenberg/assets/54422211/19a21f26-35c2-4652-a725-2cf965fe1b15

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by t-hamano** [comment]
This problem seems to occur even if you just mouse over the Navigation button on the right side of the Menu page. Perhaps it is related to the tooltip rendering.

https://github.com/WordPress/gutenberg/assets/54422211/d4f99596-bb27-467d-b571-7b3a45963382

**Comment by carolinan** [comment]
Huh I wonder if this is the same issue https://github.com/WordPress/gutenberg/issues/52680

**Comment by t-hamano** [comment]
Yes, looks like the same problem with that issue. I have submitted #52694, but it looks like there is a #52635 in progress that attempts a more fundamental solution.

**Comment by scruffian** [comment]
Fix here: https://github.com/WordPress/gutenberg/pull/52758

**Comment by t-hamano** [comment]
I have confirmed that the issue has been resolved by #52758 and will close it.
